litmus-portal/frontend/src/views/Analytics/WorkflowDashboard/WorkflowRunsBarChart/index.tsx

```diff
@@ -184,7 +184,6 @@ const WorkflowRunsBarChart: React.FC<WorkflowRunsBarChartProps> = ({
         tickwidth: 0,
         ticklen: 0,
         tickfont: {
-          family: 'Ubuntu, monospace',
```

Comment:
Where are we using this file?

Reply:
Its a Bar Chart used to workflow comparison

Reply:
we are not using this file @whitetiger1399